### PR TITLE
Allow Sylius 1.11 to 1.13

### LINKED ISSUE #189: Handle shorts Youtube link
Hello guys,

There's a new thing in Youtube called "Shorts" which is not same as Video.
But the embed is working in the same way.

Currently, the constraint validator doesn't allow us to give a shorts URL.

https://www.youtube.com/shorts/TESTEST
will become
https://www.youtube.com/embed/TESTEST

### PR DESCRIPTION
**Pull request by maximehuran**
closes #189 #228 #169

## Sylius 1.13 compatibility

image

## Youtube shorts compatibility

image

image

## Avoid error if folder `media/rich-editor` does not exists

The liip imagine folder can not exists and it will be ignored

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+| 1.12 | 8.1 - 8.2 |
+| 1.13 | 8.1 - 8.2 |
+
 ## Installation
 
+If you want to use our recipes, you can configure your composer.json by running:
+
+```json
+composer config --no-plugins --json extra.symfony.endpoint '["https://api.github.com/repos/monsieurbiz/symfony-recipes/contents/index.json?ref=flex/master","flex://defaults"]'
+```
```
Comment (etienne-monsieurbiz) [inline README.md:27]: **❔ question:** Do we really need to have this command in red in the readme ?‏

File: src/UiElement/UiElementFormOptionsTrait.php [hunk 2]
```diff
@@ -25,6 +25,7 @@ trait UiElementFormOptionsTrait
     public function getFormOptions(): array
     {
         try {
+            /** @phpstan-ignore-next-line */
```
Comment (etienne-monsieurbiz) [inline src/UiElement/UiElementFormOptionsTrait.php:28]: **🤓 nitpick:** You can ignore this error like this when it's one line return : 
`/** @var array*/`
```suggestion
            /** @var array*/
```

‏

File: src/UiElement/UiElementTrait.php [hunk 3]
```diff
@@ -14,6 +14,8 @@
 namespace MonsieurBiz\SyliusRichEditorPlugin\UiElement;
 
 use Exception;
+use PHPUnit\Framework\Assert;
```
Comment (etienne-monsieurbiz) [inline src/UiElement/UiElementTrait.php:17]: **🎯 suggestion:** It's probably not this Assert class that you want here‏

**Review by etienne-monsieurbiz — CHANGES_REQUESTED** [review]

**Review by etienne-monsieurbiz — APPROVED** [review]
LGTM

**Review by delyriand — APPROVED** [review]

**Pull request merged**